code/acting/src/acting/stanley_controller.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env python
+# from typing import List
+import math
+from math import atan, sqrt, sin, cos
+import numpy as np
+import ros_compatibility as roscomp
+from carla_msgs.msg import CarlaSpeedometer
+from geometry_msgs.msg import PoseStamped, Point  # , Pose, Quaternion
+from nav_msgs.msg import Path
+from ros_compatibility.node import CompatibleNode
+# from ros_compatibility.qos import QoSProfile, DurabilityPolicy
+from rospy import Publisher, Subscriber
+# from sensor_msgs.msg import NavSatFix, Imu
+from std_msgs.msg import Float32  # Bool
+from acting.msg import StanleyDebug
+
+from helper_functions import vector_angle
+from trajectory_interpolation import points_to_vector
+
+
+# todo: docs
```

Comment:
Docs would be desirable :)

Reply:
added docs in ea29919 and removed todo item